### ISSUE TITLE
better mobile rendering of run page

### ISSUE THREAD

**Issue by sryza** [dagster-io/dagster#5236]
Scenario: it's the weekend, you get message from someone saying your job failed. They helpfully send you a link to the run page for the failure. You open it from your phone, and you see:

![image](https://user-images.githubusercontent.com/654855/137763113-f283a3cc-c71f-4116-970b-29b03cf25273.png)

**Comment by bengotow** [comment]
Hmm cc @braunjj  @sryza  I think we should probably have a broader discussion about this one to identify what the supported goals / user stories should be on mobile. Definitely seems you should be able to use this page, but for what exactly? I could see turning off the Gantt chart entirely and focusing on logs + re-execute, or burying the logs behind a "view raw logs" button and focusing entirely on a mobile-friendly Gantt chart (potentially with subtree collapsing and some other things we've mulled about) and answering "what failed" at a glance.

**Comment by sryza** [comment]
@bengotow agreed. My take is that the #1 thing is basically: find out what step failed and the error for the failure.  I don't think it's worth putting in effort to get the gantt chart displaying nicely.  If we just showed logs, that would be a huge improvement over what we have now.

**Comment by danielgafni** [comment]
Struggling with this all the time. The assets catalog (not mentioning the lineage) is a problem too

**Comment by josh-acuitymd** [comment]
+1 to this! Was quite the struggle to find the specific asset I wanted to materialize on mobile.

**Comment by mkleinbort** [comment]
Can we please have this. Had to deal with an outage on my phone - it was a pain.

**Comment by braunjj** [comment]
Hey @mkleinbort Thanks for the feedback. We are starting to explore a revamp of the run detail page and we have a nice opportunity to make it mobile friendly in the process. What information would you want us to prioritize in the mobile version? Just Logs/Failures? Step statuses? Asset materialization statuses? 

Do you want to be able to re-execute from mobile as well?

**Comment by kevinbevers** [comment]
> Hey [@mkleinbort](https://github.com/mkleinbort) Thanks for the feedback. We are starting to explore a revamp of the run detail page and we have a nice opportunity to make it mobile friendly in the process. What information would you want us to prioritize in the mobile version? Just Logs/Failures? Step statuses? Asset materialization statuses?
> 
> Do you want to be able to re-execute from mobile as well?

If I can give some input, it would be nice to see the status of the runs in the overview, 

So name of job + runtime and background with the status: red = failed, purple = running, green = completed.
Then on mobile we could see if there are jobs hanging, need a restart or all went ok.

It's nice to enable a re-run before going to the office, since then it will be done once I arrive.

and then when going into a job I think the waterfall job list would work best, and only showing logs after some button press, since it will clutter the a mobile UI real quick.

**Comment by Jay-diehl** [comment]
I agree with Kevin, I mainly use Dagster on my phone to check whether everything ran as expected and, if not, to see what caused the error. This way, I can already tell if a retry might fix the issue before I even get to the office. To focus for me is on the things that errored. Succeed information is less important for me on the phone.

I also sometimes need to launch a run from the jobs section on my phone. However, the job names are currently hard to read, which makes it difficult to quickly find the right one.

**Comment by KvH98** [comment]
I am on the road for my job a lot. Recently, a colleague of mine who was at the office called me to check some Dagster pipelines which had failed. As the person who has most knowledge on those specific pipelines, I wanted to quickly check the error messages on the Dagster website. However, this was impossible on my mobile phone. In the end, my colleague had to send screenshots from his desktop while I was navigating him to the right assets/pipelines. This didn't work very well. With that being said, I would really appreciate an easier way to see error messages / logs on the mobile website

**Comment by mo-tech55** [comment]
Yes same here. I end up taking my laptop everywhere I go because of this

**Comment by kevinbevers** [comment]
I started an attempt at approving this using Fable 5 and an iterative approach, my aim is to do as much as possible using just CSS and only use extra mobile specific UI element when needed. 

Input regarding what would be nice to see is welcome.
